api/students/views.py

```diff
@@ -1,45 +1,19 @@
 from django.shortcuts import render
 from rest_framework import viewsets, generics
-from .models import User
-from .serializers import UserSerializer
+from .models import User, Complaint
+from .serializers import UserSerializer, ComplaintSerializer
 
 # Create your views here.
-
 class UserView(viewsets.ModelViewSet):
```

Comment:
This View also provides the endpoint for accessing and deleting individual users, so there was no need for the `UserDetailView`, so I removed it